docs/features/manual-dependencies.md

```diff
@@ -54,6 +54,7 @@ Supported dependency types:
 - `pub` - Dart dependencies found at [pub.dev](https://www.pub.dev/).
 - `pypi` - Python dependencies that are typically found at [Pypi.org](https://pypi.org/).
 - `swift` - Swift dependencies using the [Swift Package Manager](https://www.swift.org/package-manager/).
+<!-- markdown-link-check-disable-next-line -->
```

Comment:
Okay, now I'm reversing my previous comment. This comment is not on current master, and I don't think this PR should change that